src/functionalTest/resources/features/F-004/F-000.feature

```diff
@@ -0,0 +1,122 @@
+@F-004
+Feature: F-004: Post Upload Document with Binary Content
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-041
+  Scenario: must successfully Post Upload Document with Binary Content
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a positive response is received
+    And   the response [contains the metadata for the document uploaded above]
+    And   the response has all other details as expected
+
+  @S-042
```

Comment:
With in the scope of this upload API that covers only step number 2,3,4,5 &6 as per the LLD, validating the existence of casteType, JID, user-roles and user-id is not possible and hence these scenarios are out of scope.

Reply:
removed this test